test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -1916,6 +1962,27 @@ public void testGetDocContent() throws Exception {
   }
 
   @Test
+  public void testGetDocContent_noUniqueKeyFieldsInSelect() throws Exception {
```

Comment:
Just to really mess up the purpose of this CL, what about a test that uses the wrong column in singleDocContentSql, just to clarify that we can't match column names when substituting parameters. That is, have 2 rows, with id = 1, notId = 0, and id = 2, notId = 1, and use a docId value of 1 but with notId = ? in the query. It doesn't test anything in our code, it just confirms intent and functionality in the way the queries are written.